otx/cli/utils/tests.py

```diff
@@ -84,6 +84,37 @@ def otx_train_testing(template, root, otx_dir, args):
     assert os.path.exists(f"{template_work_dir}/trained_{template.model_template_id}/label_schema.json")
 
 
+def otx_resume_testing(template, root, otx_dir, args):
```

Comment:
how about adding `"--resume-from",` in `otx_train_testing` and reusing that function instead make a new function?

Reply:
If we are okay to overwrite trained weight with resumed weight, we can merge in one function (`ote_train_testing`).
It was inevitable to avoid checkpoint overwritting. (`save_model_to` argument should be updated.)